app/components/wallet/send/WalletSendForm.js

```diff
@@ -221,25 +221,21 @@ export default class WalletSendForm extends Component<Props, State> {
     const amountFieldProps = amountField.bind();
     const totalAmount = formattedAmountToBigNumber(amountFieldProps.value).add(transactionFee);
 
-    const componentClasses = classicTheme ? styles.componentClassic : styles.component;
-    const receiverInputClasses = classicTheme ? styles.receiverInputClassic : styles.receiverInput;
-    const amountInputClasses = classicTheme ? styles.amountInputClassic : styles.amountInput;
-
-    const hasPendingTxWarning = (
+    const pendingTxWarningComponent = (
       <div className={styles.contentWarning}>
         <SvgInline svg={dangerIcon} className={styles.icon} />
         <p className={styles.warning}>{intl.formatMessage(messages.sendingIsDisabled)}</p>
       </div>
     );
 
     return (
-      <div className={componentClasses}>
+      <div className={styles.component}>
 
-        {hasAnyPending && hasPendingTxWarning}
+        {hasAnyPending && pendingTxWarningComponent}
```

Comment:
this rename is additional in this PR, `hasPendingTxWarning` feels like boolean item